.github/workflows/nodejs.yml

```diff
@@ -3,48 +3,47 @@ name: Node CI
 on:
   push:
     branches:
-    - master
+      - master
   pull_request:
     branches:
-    - master
+      - master
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
-        node-version: [14.x]
+        node-version: [14, 15, 16]
```

Comment:
we don't really need 15 here, as odd-numbered releases become unsupported after 6 months

Reply:
Correct,I just wanted to make sure we're 100% covered.